runtime/drivers/sqlite/migrations/0026.sql

```diff
@@ -0,0 +1,8 @@
+PRAGMA foreign_keys = ON;
+
+CREATE TABLE IF NOT EXISTS instance_health (
+    instance_id TEXT PRIMARY KEY,
+    health BLOB NOT NULL,
+    created_on TIMESTAMP NOT NULL,
```

Comment:
Should this be named `updated_on`? Since it changes when the health is updated. (It probably doesn't make sense to have a separate `created_on` field as it would just be roughly the same as that of the instance in the `instances` table.)

Reply:
Yes. Forgot to update the db column names here. Thanks.